lib/routes/universities/cug/gcxy/index.js

```diff
@@ -70,40 +63,40 @@ module.exports = async (ctx) => {
                 },
             });
             const $ = cheerio.load(response.data);
-            const items = $('div.right_list > ul > li');
+            const rowList = $('ul.col-news-list > li.list_item');
+            // get all items from the rowList
+            const items = rowList
+                .map((index, item) => {
+                    const li = $(item);
+                    return {
+                        title: li.find('a.news-title').text(),
+                        link: new URL(li.find('a.news-title').attr('href'), host).href,
+                        description: '',
```

Comment:
```suggestion
```

Reply:
Please explain how setting an empty property `description: ''` can help converting relative path to absolute path.